### PR TITLE
Add logging handler [RHELDST-3456]

### PR DESCRIPTION
**Pull request by nathanegillett**
Resolves an issue wherein log messages weren't output.

### CODE REVIEW

File: exodus_gw/main.py [hunk 1]
```diff
@@ -34,3 +34,13 @@ async def custom_http_exception_handler(request, exc):
...
+
+    root = logging.getLogger()
+    if not root.hasHandlers():
+        fmtr = logging.Formatter(
+            fmt="[%(asctime)s] [%(process)s] [%(levelname)s] %(message)s",
+            datefmt="%Y-%m-%d %H:%M:%S %z",
+        )
+        hdlr = logging.StreamHandler()
+        hdlr.setFormatter(fmtr)
+        root.addHandler(hdlr)
```
Comment (rohanpm) [inline exodus_gw/main.py:46]: A question: in the case where uvicorn logger already has a handler (which right now seems unclear when that does or does not happen), does the added handler here result in duplicate messages?

**Review by rohanpm — APPROVED** [review]
A small test of the form "after configure_loggers is called, root logger has at least one handler" would be nice too.

**Review by rohanpm — CHANGES_REQUESTED** [review]
> A small test of the form "after configure_loggers is called, root logger has at least one handler" would be nice too.

Actually I missed that the coverage had dropped from 100%.  I don't want that, so putting it more strongly - there should be such a test, of the form "if root logger had no handlers before we called configure_loggers, it has a handler afterward".

**Review by crungehottman — APPROVED** [review]

File: exodus_gw/main.py (hunk 1 above)
Comment (nathanegillett) [reply exodus_gw/main.py:46]: **gunicorn**:
> $ gunicorn -k uvicorn.workers.UvicornWorker --bind 0.0.0.0:8080 exodus_gw.main:app --reload
[2020-11-10 09:51:23 -0500] [7805] [INFO] Starting gunicorn 20.0.4
[2020-11-10 09:51:23 -0500] [7805] [INFO] Listening at: http://0.0.0.0:8080 (7805)
[2020-11-10 09:51:23 -0500] [7805] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2020-11-10 09:51:23 -0500] [7808] [INFO] Booting worker with pid: 7808
[2020-11-10 09:51:23 -0500] [7808] [INFO] Started server process [7808]
[2020-11-10 09:51:23 -0500] [7808] [INFO] Waiting for application startup.
[2020-11-10 09:51:23 -0500] [7808] [INFO] root logger at expected level
[2020-11-10 09:51:23 -0500] [7808] [DEBUG] s3 logger at expected level
[2020-11-10 09:51:23 -0500] [7808] [INFO] Application startup complete.

**uvicorn**:
> $ uvicorn exodus_gw.main:app --reload
INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
INFO:     Started reloader process [8480] using watchgod
INFO:     Started server process [8482]
INFO:     Waiting for application startup.
[2020-11-10 09:58:42 -0500] [8482] [INFO] root logger at expected level
[2020-11-10 09:58:42 -0500] [8482] [DEBUG] s3 logger at expected level
INFO:     Application startup complete.
[2020-11-10 09:58:42 -0500] [8482] [INFO] Application startup complete.
[2020-11-10 09:58:45 -0500] [8482] [INFO] ran healthcheck
INFO:     127.0.0.1:47970 - "GET /healthcheck HTTP/1.1" 200 OK

Looks like uvicorn's log messages are emitted by the new handler but not the other way around. I don't think it'll be an issue, though, as the first example (gunicorn) is the "correct" way to start the application.

**Review by rohanpm — APPROVED** [review]

**Pull request merged**